presto-hive/src/main/java/com/facebook/presto/hive/HiveMetadata.java

```diff
@@ -1651,6 +1655,11 @@ public HiveOutputTableHandle beginCreateTable(ConnectorSession session, Connecto
                         .collect(Collectors.toList())));
     }
 
+    public static boolean createFilesForMissingBuckets(Table table)
```

Comment:
nit: `shouldCreateFilesForMissingBuckets`? The current method name is more like an action.

Reply:
Done